_data/BezierSurgeon.yml

```diff
@@ -0,0 +1,9 @@
+description: An extension for precision point insertion.
+developer: Connor Davenport
+developerURL: http://www.connordavenport.com
+extensionName: BezierSurgeon
+extensionPath: BezierSurgeon.roboFontExt
+repository: https://github.com/connordavenport/BezierSurgeon
+icon: https://raw.githubusercontent.com/connordavenport/BezierSurgeon/master/images/mechanicIcon.png
+tags: [interpolation, drawing]
+dateAdded: 2019-06-15 16:03:24
```

Comment:
could you change the date to today :)

Reply:
Oops sorry, I copied the Theme Manager file😜 fixing now

Reply:
super!